domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewScheduleDateCalculationService.kt

```diff
@@ -109,6 +109,10 @@ class ReviewScheduleDateCalculationService {
    * Review Schedule should not get an adjusted due date. This is to help prevent a Review being extended repeatedly to
    * prevent it becoming overdue by repeatedly exempting and clearing the exemption.
    *
+   * The exception to this is when the existing Review Schedule Status is EXEMPT_PRISONER_TRANSFER. In this case we
```

Comment:
I needed to add this to the kdoc for the method, because the impl has changed at some point - we now have code right at the top of the method that sets today+10 for transfers regardless of the previous date
(it confused me initially when I was working out where I needed to make the changes for this PR ! 🤣 )